2 - RTB/Documentazione esterna/Analisi dei Requisiti/Analisi dei Requisiti.typ

```diff
@@ -269,27 +269,24 @@ $bold("Postcondizioni: ")$
 $bold("Scenario: ")$
 - L'utente ha caricato un file per la configurazione dell'ambiente contenente informazioni incongruenti.
 
-== Creazione manuale della mappa del magazzino
+== Creazione magazzino vuoto
 $bold("Descrizione: ")$
-configurazione manuale del perimetro dell'ambiente di lavoro.
+All'avvio dell'applicazione e in ogni momento si desideri, si può decidere di creare un ambiente di lavoro vuoto di dimensioni predefinite.
 
 $bold("Attore: ")$
 utente.
 
 $bold("Precondizioni: ")$
-- è stato dato inizio alla procedura di configurazione manuale dell'ambiente di lavoro.
+- è stato dato inizio alla procedura di creazione dell'ambiente di lavoro vuoto.
 
 $bold("Postcondizioni: ")$
-- la forma e il perimetro dell'ambiente di lavoro è stato configurato manualmente;
+- è stato generato un ambiente di lavoro vuoto di dimensioni standard;
```

Comment:
standard -> predefinite. Standard è un termine tecnico che stabilisce che tali dimensioni provengono da uno standard. In questo contesto è meglio definirle come "dimensioni predefinite"